### DISCUSSION TITLE
Deposit boxes don't work

### DISCUSSION THREAD

**Discussion by CnPx98** [nofatigue/runelite-profit-tracker#26] [Ideas]
Can you please add deposit boxes? When you use them it sees you as dropping the items rather then banking them. So rather than the tracker sticking when ever you deposit an item it subtracts it from your total.

**Comment by doughboy623** [comment]
definitely would be great i want to know my profit or an idea but hard to tell when i don't wanna waste the extra time opening bank chest rather then deposit box. Please add this!
